scripts/podspec_bump_version.rb

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env ruby
+
+# Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2019-Present Datadog, Inc.
+
+require 'cocoapods'
+
+version = ARGV[0]
+if !version
+    raise "Usage: podspec_bump_version.rb <version number>"
+end
+
+podspecs = Dir.glob('*.podspec')
+
+for podspec in podspecs do
+    spec = Pod::Specification.from_file(podspec)
+    text = File.read(podspec)
+    text.gsub!(/(s.version( )*= ")#{spec.version}(")/, "\\1#{version}\\3")
```

Comment:
After all, we just do string replacement (similar to previous `sed///g`), so I wonder if it's worth introducing Ruby complexity for this PR. What do we benefit from `spec = Pod::Specification.from_file(podspec)` 🤔?

Wouldn't simple regex for matching:
```
  s.version      = "([a-z][A-Z][0-9]\.\-)*"
```
be enough?

Separate thing is that we need `Bundler` anyway as we introduced `Cocoapods` dedpendency in #1171 👍

Reply:
That was a good suggestion!! I was able to do the same with a shell script using `find` and `sed`!
I've left the Gemfile and Bundler setup so we have it available, it's no longer related to this PR but as you mention we need cocoapods as a dependency in v2